lib/openstack/openstack_handle/handle.rb

```diff
@@ -53,7 +55,13 @@ def self.raw_connect(username, password, auth_url, service = "Compute", extra_op
       # throw an error trying to build an connection error message.
       opts[:openstack_service_type] = ["object-store"] if service == "Storage"
 
-      Fog.const_get(service).new(opts)
+      if service == "Planning"
+        # Special behaviour for Planning service Tuskar, since it is OpenStack specific service, there is no
+        # Fog::Planning module, only Fog::OpenStack::Planning
+        Fog::Openstack.const_get(service).new(opts)
```

Comment:
@blomquisg I think not, the others are using OpenStack, for some reason only Openstack namespace works for pblaho. I haven't investigated that further. 

Nevertheless others are like Fog::Compute::Openstack. Planning is special, because it is missing the fog-core namespace Fog::Planning